packages/protocell/protocell.1.0.0/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+homepage: "https://github.com/martinslota/protocell/"
+dev-repo: "git+https://github.com/martinslota/protocell.git"
+bug-reports: "https://github.com/martinslota/protocell/issues"
+doc: "https://martinslota.github.io/protocell/"
+license: "Apache-2.0"
+maintainer: "Martin Slota <martin.slota@keemail.me>"
+build: [["dune" "build" "-p" name "-j" jobs]]
+run-test: [["dune" "runtest" "-p" name "-j" jobs]]
+depexts: [
```

Comment:
It may be worth extracting this into a `conf-protobuf` in the future. I don't think it's a problem for now

Reply:
There's already a recently created `conf-protoc` but it doesn't actually install the `-dev` dependencies which is a problem here.

Going forward I'll try to figure out whether it is better to release a new version of `conf-protoc` or make a branch new package.